src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Car get(Long id);
```

Comment:
get in dao should return optional

Reply:
I will do it. But by the conditions of the HW 04: ...
CarService methods:
...
- Car get(Long id);